Makefile

```
@@ -309,6 +333,17 @@ define create-local-cluster
 	--agents 1 --no-lb --k3s-server-arg "--no-deploy=traefik,servicelb,metrics-server" --network $(CLUSTER_GSLB_NETWORK)
 endef
 
+define deploy-k8gb-helm
```

Consider renaming:
```suggestion
define deploy-k8gb-with-helm
```
(in analogy with `wait-for-ingress`)